jax/_src/numpy/lax_numpy.py

```diff
@@ -442,8 +442,39 @@ def fmax(x1: ArrayLike, x2: ArrayLike) -> Array:
 def issubdtype(arg1: DTypeLike, arg2: DTypeLike) -> bool:
   return dtypes.issubdtype(arg1, arg2)
 
-@util.implements(np.isscalar)
 def isscalar(element: Any) -> bool:
+  """Check if an element is a scalar.
+
+  JAX implementation of :func:`numpy.isscalar`.
+
+  The JAX version has a few differences from the NumPy version:
+
+  - In NumPy, scalars and zero-dimensional arrays are distinct.
```

Comment:
I think these bullet points are too abstract to be useful. Someone without any background knowledge on NumPy's object structure will have no idea what is meant by "scalars and zero-dimensional arrays are distinct," and what they should expect from `np.isscalar` based on this sentence.